rotkehlchen/data/importer.py

```diff
@@ -469,3 +480,269 @@ def import_cryptocom_csv(self, filepath: Path) -> Tuple[bool, str]:
                 except KeyError as e:
                     return False, str(e)
         return True, ''
+
+    def _consume_blockfi_entry(self, csv_row: Dict[str, Any]) -> None:
```

Comment:
Please add a docstring with what this does AND importantly which exceptions it may raise